extensions/amp-story/1.0/_locales/en.json

```diff
@@ -250,5 +250,13 @@
   "100": {
     "string": "Read more",
     "description": "Label for a button to open a drawer containing additional content."
+  },
+  "101": {
+    "string": "Shop",
+    "description": "Label for the CTA page outlink for shopping component when one shopping tag is present."
```

Comment:
Concise is better.

"Label for the shopping CTA button when one shopping tag is present."